.github/workflows/metrics.yml

```diff
@@ -91,3 +91,25 @@ jobs:
           config: ./metrics/metrics-${{ matrix.platform }}.yml
           sauce-user: ${{ secrets.SAUCE_USERNAME }}
           sauce-key: ${{ secrets.SAUCE_ACCESS_KEY }}
+
+  metrics-dart:
+    name: Console
+    runs-on: macos-latest
```

Comment:
any specific reason why we run this on macos?

Reply:
No particular reason. Would it be beneficial to run on ubuntu?